config/rbac/default_lmeval_user_rolebinding.yaml

```diff
@@ -0,0 +1,19 @@
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRoleBinding
+metadata:
+  name: default-lmeval-user-rolebinding
+  labels:
+    app.kubernetes.io/name: clusterrolebinding
+    app.kubernetes.io/instance: default-lmeval-user-rolebinding
+    app.kubernetes.io/component: rbac
+    app.kubernetes.io/created-by: trustyai-service-operator
+    app.kubernetes.io/part-of: trustyai-service-operator
+    app.kubernetes.io/managed-by: kustomize
+roleRef:
+  apiGroup: rbac.authorization.k8s.io
+  kind: ClusterRole
+  name: lmeval-user-role
```

Comment:
not sure i understand this:
both clusterrolebinding:  `default-lmeval-user-rolebinding` and `lmeval-user-rolebinding`
do the same thing: connect ClusterRole` lmeval-user-role` and login user `system:authenticated`
isnt one of them enough to grant login user wht * permission on lmevaljobs ?

Reply:
Thanks @zdtsw! You're absolutely right, the default LMEval rolebinding is enough, no need for a "user" rolebinding (I've removed `config/rbac/non_admin_lmeval_rolebinding.yaml`).